.github/workflows/docker-release.yaml

```diff
@@ -0,0 +1,46 @@
+name: docker_release
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
This adds a main tag to match the branch. It's useful for testing merged but unreleased changes.